bindings/pyroot/pythonizations/test/CMakeLists.txt

```diff
@@ -187,11 +187,10 @@ if (dataframe)
     # std::string_view in CPyCppyy
     ROOT_ADD_PYUNITTEST(pyroot_string_view string_view.py)
     if(NOT MSVC OR win_broken_tests)
-        if(NOT (DEFINED ENV{ROOTTEST_IGNORE_NUMBA_PY2} AND PYTHON_VERSION_MAJOR_Development_Main EQUAL 2) AND NOT
-            (DEFINED ENV{ROOTTEST_IGNORE_NUMBA_PY3} AND PYTHON_VERSION_MAJOR_Development_Main EQUAL 3))
+        if(NOT (DEFINED ENV{ROOTTEST_IGNORE_NUMBA_PY3} AND PYTHON_VERSION_MAJOR EQUAL 3))
             # Test wrapping Python callables for use in C++ using numba
             ROOT_ADD_PYUNITTEST(pyroot_numbadeclare numbadeclare.py PYTHON_DEPS numba)
-            if(PYTHON_VERSION_STRING_Development_Main VERSION_GREATER_EQUAL 3.7)
+            if(PYTHON_VERSION_STRING VERSION_GREATER_EQUAL 3.7)
```

Comment:
You want to do the `python=>3.8` requirement in a separate PR? Just to make sure that this was no oversight.